operator/deploy/deploy-kiali-operator.sh

```diff
@@ -892,7 +893,7 @@ if [ "${AUTH_STRATEGY}" == "" ]; then
 fi
 
 # Verify the AUTH_STRATEGY is a proper known value
-if [ "${AUTH_STRATEGY}" != "login" ] && [ "${AUTH_STRATEGY}" != "openshift" ] && [ "${AUTH_STRATEGY}" != "anonymous" ]; then
+if [ "${AUTH_STRATEGY}" != "login" ] && [ "${AUTH_STRATEGY}" != "openshift" ] && [ "${AUTH_STRATEGY}" != "anonymous" && [ "${AUTH_STRATEGY}" != "ldap" ]; then
```

Comment:
If I select auth_strategy of `ldap` at the prompt and this script installs Kiali, remember this will install Kiali with all default settings for LDAP. I don't think Kiali will work (or even start properly) without the ability for the user to configure the LDAP settings in auth.ldap. This is why I said in a previous comment that for now we can probably just leave this script file as-is - if you want to install with LDAP enabled, the user can write a kiali CR yaml and pass it via `--kiali-cr`. See my comment regarding this here: https://github.com/kiali/kiali/pull/1253#pullrequestreview-263658729
So I would revert all changes to this deploy-kiali-operator.sh script and do not change it in this PR (we can enhance this script in the future by allowing the user to pass in those additional LDAP settings, but that is going to complicated to do so let's not do it for this PR and just do it at a later time in a future PR).